_maps/map_files/Pahrump-Sunset/RockSprings.dmm

```diff
@@ -542,6 +542,20 @@
 	icon_state = "dirt"
 	},
 /area/f13/wasteland/rocksprings)
+"avK" = (
+/obj/machinery/camera{
+	invisibility = 1;
+	vis_flags = 0;
+	icon = 'icons/fallout/turfs/asphalt.dmi';
+	icon_state = "horizontalinnermain0";
+	name = "\proper road";
```

Comment:
This clearly doesn't do whatever you think it does. If you hit it, it'll say "you hit road" instead of "you hit the road." Just remove the `\proper`.